plugins/ocm/CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@ To start a development setup in isolation with a faster setup and hot reloads, c
 1. Run the `ocm-backend` plugin in the `plugins/ocm-backend` directory by executing the following command:
 
    ```console
-   yarn start
+   LEGACY_BACKEND_START=true yarn start
```

Comment:
This change doesn't seem necessary, unless I am missing something as I was able to start the OCM plugin by just running `yarn start`. Albeit, I didn't test any of the functionality. Did you seen any difference in functionality when testing?

Reply:
I added this because it stuck on `$ backstage-cli package start` for me and didn't start the backend.
If you run `yarn start` inside `ocm-backend`, does the backend start for you (`backstage info Listening on :7007`)?
To check if backend runs, you can try: `curl -X GET http://localhost:7007/api/ocm/status`.

Reply:
No, I needed `LEGACY_BACKEND_START=true` for the backend. It wouldn't start properly without. This comment was just meant for the frontend plugin only.

Reply:
Oh, I see. This is instruction also for backend. I needed to change this twice: once for instructions in ocm package for ocm-backend (this change) and once for instructions in ocm-backend for ocm-backend.

Reply:
Sorry, I completely missed that part. Thought this was for the frontend.